samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -20,13 +20,22 @@
 			reg = <0x100000 DT_SIZE_K(908)>;
 		};
 
-		storage_partition: partition@1e3000 {
-			reg = <0x1e3000 DT_SIZE_K(20)>;
-		};
+		/* Leaving here an empty gap from 0x1e3000 to 0x1e3fb0.
+		 * We want to have a bt_fast_pair_partition preceding the
+		 * storage_partition. We also need a storage partition to
+		 * start at address divisible by 0x1000, because this is the
+		 * storage sector size. What is more, we want to have all
+		 * partition sizes and addresses divisible by 0x10 which is
+		 * the write block size for MRAM.
+		 */
```

Comment:
Consider: TBH I personaly would leave only the information that there is the gap between this and that address.

I do not think that it is requirement to have bt_fast_pair_partition before the settings partition and it is only our "style" i guess to be aligned with other apps, right?
And the rest is just how the storage / MRAM works.

If needed we could add it in commit message i guess :D

Reply:
I think it has educational value to have it here and explain what is happening.